# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/webcodecs/video-decoder.https.any.js.ini
# Commit: aea093a94985
# Full Hash: aea093a949856e14212082873768c90e664fb595
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2023-06-30 04:06:43
# Regressor Bug: 1831451
# File Overlap Count: 1
# Description:
#   Bug 1831451 - Implement VideoDecoder::Configure r=padenot
#   
#   This patch implements configure method [1] for WebCodecs' VideoDecoder
#   interface. The configure job is via DecoderAgent's Configure().
#   
# ==============================================================================

diff -r 13f8ed81dca8 -r aea093a94985 testing/web-platform/meta/webcodecs/video-decoder.https.any.js.ini
--- a/testing/web-platform/meta/webcodecs/video-decoder.https.any.js.ini	Thu Jun 29 22:33:12 2023 +0000
+++ b/testing/web-platform/meta/webcodecs/video-decoder.https.any.js.ini	Thu Jun 29 22:33:12 2023 +0000
@@ -3,44 +3,8 @@
   expected:
     if (os == "android") and fission: [OK, TIMEOUT]
 
-  [Test that VideoDecoder.configure() rejects invalid config:Empty codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.configure() rejects invalid config:Unrecognized codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.configure() rejects invalid config:Audio codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.configure() rejects invalid config:Ambiguous codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.configure() rejects invalid config:Codec with MIME type]
-    expected: FAIL
-
-  [Test VideoDecoder construction]
-    expected: FAIL
-
 
 [video-decoder.https.any.worker.html]
   prefs: [dom.media.webcodecs.enabled:true]
   expected:
     if (os == "android") and fission: [OK, TIMEOUT]
-
-  [Test that VideoDecoder.configure() rejects invalid config:Empty codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.configure() rejects invalid config:Unrecognized codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.configure() rejects invalid config:Audio codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.configure() rejects invalid config:Ambiguous codec]
-    expected: FAIL
-
-  [Test that VideoDecoder.configure() rejects invalid config:Codec with MIME type]
-    expected: FAIL
-
-  [Test VideoDecoder construction]
-    expected: FAIL